int32 dummy
---
string[] names
vision_msgs/BoundingBox3D[] obstacles